tcms/testplans/static/testplans/js/get.js

```diff
@@ -0,0 +1,140 @@
+const TestPlanModule = (function() {
+    'use strict';
+
+    var testplan_id = null,
+        test_cases = {
+            'confirmed': [],
+            'not_confirmed': []
+        };
+
+    var Constructor = function() {
```

Comment:
out of curiosity: is there a reason to use `var`, instead of `let` or `const`?

Reply:
I would use let and const only with ES6 syntax and with some JavaScript compiler(Babel).Also I do not need different scope that "let" provides.I know we use const in some places, so I only used it for the module variable declaration.